src/Compilers/CSharp/Portable/Syntax/Syntax.xml

```diff
@@ -1716,33 +1716,29 @@
   <Node Name="PropertyPatternSyntax" Base="PatternSyntax">
     <Kind Name="PropertyPattern"/>
     <Field Name="Type" Type="TypeSyntax" />
-    <Field Name="PatternList" Type="SubPropertyPatternListSyntax" />
-  </Node>
-  <Node Name="SubPropertyPatternListSyntax" Base="CSharpSyntaxNode">
-    <Kind Name="SubPropertyPatternList"/>
     <Field Name="OpenBraceToken" Type="SyntaxToken">
       <Kind Name="OpenBraceToken"/>
       <PropertyComment>
-        <summary>SyntaxToken representing open brace.</summary>
+        <summary>SyntaxToken representing the open brace.</summary>
+      </PropertyComment>
+    </Field>
+    <Field Name="Expressions" Type="SeparatedSyntaxList&lt;ExpressionSyntax&gt;">
```

Comment:
Also, I think we should use more specific type for the list element, i.e. IsPatternExpressionSyntax. It looks like BindSubPropertyPatterns doesn't expect any other type.